content/en/docs/concepts/services-networking/connect-applications-service.md

```diff
@@ -314,8 +314,11 @@ kubectl delete deployments,svc my-nginx; kubectl create -f ./nginx-secure-app.ya
 At this point you can reach the nginx server from any node.
 
 ```shell
-kubectl get pods -o yaml | grep -i podip
+kubectl get pods -l run=my-nginx -o yaml | grep -i podip
```

Comment:
Consider changing this to not use a `grep`; `kubectl` has native abilities to help show you a particular field.

Reply:
I'd love to do so however I also believe @chetak123 is already working on a PR for this as indicated in https://github.com/kubernetes/website/issues/34388#issuecomment-1159556864